syntax = "proto3";

package million.ss.test;

import "google/protobuf/descriptor.proto";
import "ss/ss_msgid.proto";

enum SubMsgId {
    option (million.ss.msg_id) = MSG_ID_TEST;
    
    SUB_MSG_ID_INVALID = 0;
    SUB_MSG_ID_LOGIN_REQ = 1;
    SUB_MSG_ID_LOGIN_RES = 2;
    
}

extend google.protobuf.MessageOptions {
    SubMsgId sub_msg_id = 11001;
}

message LoginReq {
    option (sub_msg_id) = SUB_MSG_ID_LOGIN_REQ;
    string value = 1;
}

message LoginRes {
    option (sub_msg_id) = SUB_MSG_ID_LOGIN_RES;
    string value = 1;
}